### PR TITLE
Update set-up-a-custom-blocked-urls-list-atp.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note to the document stating that only 500 URLs can be added to the block list.

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2467

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/office-365-security/set-up-a-custom-blocked-urls-list-atp.md [hunk 1]
```diff
@@ -80,6 +80,10 @@ While you add URLs to your list, keep the following points in mind:
 |`https://contoso.com/a*`|Blocks a site `https://contoso.com/a` and additional subpaths like `https://contoso.com/a/b`|
 |`https://toys.contoso.com*`|Blocks a subdomain ("toys" in this case) but allow clicks to other domain URLs (like `https://contoso.com` or `https://home.contoso.com`).|
 
+
+> [!Note]
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/set-up-a-custom-blocked-urls-list-atp.md]:
```suggestion
> [!NOTE]
```

File: microsoft-365/security/office-365-security/set-up-a-custom-blocked-urls-list-atp.md [hunk 2]
```diff
@@ -80,6 +80,10 @@ While you add URLs to your list, keep the following points in mind:
...
+> By default, You can only add 500 URLs to the blocked URL list in the Office 365 ATP Safe Links default policy.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/set-up-a-custom-blocked-urls-list-atp.md]:
```suggestion
> By default, you can only add 500 URLs to the blocked URL list in the Office 365 ATP Safe Links default policy.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**